File: server/AppSupport.Data/Entities/ManifestPerson.cs
namespace AppSupport.Data.Entities
{
    public class ManifestPerson
    {
        public int Id { get; set; }
        public int ManifestPlaneId { get; set; }
        public int OrganizationId { get; set; }
        public int PersonId { get; set; }
        public int RankId { get; set; }
        public int TravelerId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MiddleName { get; set; }
        public string Nickname { get; set; }
        public string Occupation { get; set; }
        public string Title { get; set; }

        public ManifestPlane ManifestPlane { get; set; }
        public Organization Organization { get; set; }
        public Person Person { get; set; }
        public Rank Rank { get; set; }
        public Person Traveler { get; set; }
    }
}

File: client/core/api/models/manifest-person.ts
import { ManifestPlane } from './manifest-plane';
import { Organization } from './organization';
import { Person } from './person';
import { Rank } from './rank';

export interface ManifestPerson {
  id: number;
  manifestPlaneId: number;
  organizationId: number;
  personId: number;
  rankId: number;
  travelerId: number;
  firstName: string;
  lastName: string;
  middleName: string;
  nickname: string;
  occupation: string;
  title: string;

  manifestPlane: ManifestPlane;
  organization: Organization;
  person: Person;
  rank: Rank;
  traveler: Person;
}
